modules/xmpp/JingleSessionPC.js

```diff
@@ -510,8 +511,44 @@ export default class JingleSessionPC extends JingleSession {
                 break;
             }
         };
+
+        /**
+         * The negotiationneeded event is fired whenever we shake the media on the
+         * RTCPeerConnection object.
+         */
         this.peerconnection.onnegotiationneeded = () => {
+            const state = this.peerconnection.signalingState;
+            const remoteDescription = this.peerconnection.remoteDescription;
+
             this.room.eventEmitter.emit(XMPPEvents.PEERCONNECTION_READY, this);
+            if (browser.usesUnifiedPlan() && state.toString() === 'stable'
```

Comment:
is the toString() necesssary?

Reply:
I can't recall why I added that check but looks like its not necessary.